packages/manager/apps/dedicated/client/app/dedicated/server/details/server.controller.js

```diff
@@ -220,6 +220,11 @@ export default class ServerCtrl {
       this.loadServer();
     });
 
+    this.$scope.$on('dedicated.informations.update.monitoring', () => {
+      this.server.monitored = !this.server.monitored;
```

Comment:
how are you sure that the monitoring value is toggled ?
it's possible to call updateMonitoring method with the same value, in that case monitoring won't change

i think it would be safer to pass the monitoring value as an argument of this event so you can write
```js
this.$scope.$on('dedicated.informations.update.monitoring', (isMonitored) => {
// ...
```